.gitignore

```diff
@@ -12,9 +12,12 @@
 *private_key*
 keys.json
 
+data/
 agent/
 service/
 
+asylum_abci_app/
```

Comment:
Should be created as part of generate .gitignore.

Lets ensure this is in auto dev,.